Automattic/jetpack
# Podcast Player: Refactor front-end icons

#### Changes proposed in this Pull Request:

Brings back the Player's icons (React-style) after a refactor done in the https://github.com/Automattic/jetpack/pull/15072.

#### Testing instructions:

* Add a Podcast Player block to your page
* Initialize the Player (you can use this URL: https://anchor.fm/s/9400d7c/podcast/rss)
* Open the page preview.
* Click one of the episodes to play it - a loudspeaker icon with text color fill should appear
* Click again the same episode to pause it - the icon should disappear
* Block one of the episode audio URLs to test the error icon
* Click the episode you just blocked - a red exclamation mark icon should appear.
* Click any other episode - the icon should disappear as the error message does.

#### Example screenshots:

| Episode Playing  | Episode Error |
| ------------- | ------------- |
| <img width="631" alt="Screenshot 2020-03-24 17 25 38" src="https://user-images.githubusercontent.com/1451471/77451215-ac657700-6df4-11ea-80b1-a317e244a7bd.png">  | <img width="630" alt="Screenshot 2020-03-24 17 25 23" src="https://user-images.githubusercontent.com/1451471/77451207-aa031d00-6df4-11ea-8fd8-041587c0863a.png">  |


#### Proposed changelog entry

* Podcast Player: Refactor front-end icons react-style
